src/components/containers/Sidebar/Sidebar.css

```diff
@@ -1,9 +1,31 @@
 .sidebar {
-  /* background-color: cadetblue; */
-  width: 30%;
+  width: 40%;
   align-items: center;
-  background-color: lightgrey;
-  border-radius: 1rem;
-  height: 35rem;
-  margin: 2rem;
+  height: 10rem;
+  margin: 1rem;
+  padding: 5px;
+}
+
+.sidebar:hover {
```

Comment:
Do we need this? To my eyes, this bit of css will not do anything.

Reply:
This we do need. We can change the padding, but everything else is needed.